app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -4,6 +4,9 @@ module VerifyProfileConcern
   def url_for_pending_profile_reason
     return idv_verify_by_mail_enter_code_url if current_user.gpo_verification_pending_profile?
     return idv_in_person_ready_to_verify_url if current_user.in_person_pending_profile?
+    # We don't want to hit idv_please_call_url in cases where the user
+    # has fraud review pending and not passed at the post office
```

Comment:
nit: Can we be more specific with "fraud review pending"? I was thinking something like ` tmx fraud review pending` ( or txm fraud review in review or reject).